Sources/ClientRuntime/Networking/Http/URLSession/URLSessionConfiguration+HTTPClientConfiguration.swift

```diff
@@ -13,8 +13,8 @@ extension URLSessionConfiguration {
 
     public static func from(httpClientConfiguration: HttpClientConfiguration) -> URLSessionConfiguration {
         let config = URLSessionConfiguration.default
-        if let connectTimeout = httpClientConfiguration.connectTimeout {
-            config.timeoutIntervalForRequest = connectTimeout
+        if let socketTimeout = httpClientConfiguration.socketTimeout {
+            config.timeoutIntervalForRequest = socketTimeout
```

Comment:
See my note above.  How we transfer the HTTP config fields to specific clients should be made clear in the doc comments.

Reply:
Both config fields are now used for same purposes in both clients, so I think existing comments do their job without exposing arbitrary details.